src/Microsoft.VisualStudio.Services.Agent/JobServer.cs

```diff
@@ -173,7 +173,13 @@ public async Task<(DedupIdentifier dedupId, ulong length)> UploadAttachmentToBlo
         {
             int maxParallelism = HostContext.GetService<IConfigurationStore>().GetSettings().MaxDedupParallelism;
             var (dedupClient, clientTelemetry) = await DedupManifestArtifactClientFactory.Instance
-                    .CreateDedupClientAsync(verbose, (str) => Trace.Info(str), this._connection, maxParallelism, cancellationToken);
+                    .CreateDedupClientAsync(
+                        verbose,
+                        (str) => Trace.Info(str),
+                        this._connection,
+                        maxParallelism,
+                        null,
```

Comment:
nit: can you make this named parameter, agent:null, it isn't super clear why we are passing null here.

Reply:
done